deps/pom.xml

```diff
@@ -39,6 +39,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <junit-version>4.12</junit-version>
         <mockito-version>1.10.19</mockito-version>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
Will remove this before merge, this is to correct a few error messages on mvn install relating to docs missing for some methods.